### PR TITLE
[ETFE-1502] GetMovementResponse change

### PR DESCRIPTION
**Pull request by grahampaulcook**

### CODE REVIEW

File: test/uk/gov/hmrc/emcstfe/models/response/GetMovementResponseSpec.scala [hunk 1]
```diff
@@ -903,143 +903,143 @@ class GetMovementResponseSpec extends UnitSpec with GetMovementFixture {
...
-            |						</body:BodyEadEsad>
-            |					</body:EADESADContainer>
-            |				</body:Body>
-            |			</body:IE801>
-            |		</mov:currentMovement>
-            |	</mov:movementView>""".stripMargin)
-
-        GetMovementResponse.xmlReader.read(noConsignorXML) shouldBe ParseFailure(EmptyError(GetMovementResponse.consignorName))
-      }
+//      "missing consignorName" in {
```
Comment (MJCallahanPage) [inline test/uk/gov/hmrc/emcstfe/models/response/GetMovementResponseSpec.scala:906]: Are these tests no possible now? Or do they still need to be updated?

**Review by MJCallahanPage — COMMENTED** [review]
Just one query

File: test/uk/gov/hmrc/emcstfe/models/response/GetMovementResponseSpec.scala (hunk 1 above)
Comment (grahampaulcook) [reply test/uk/gov/hmrc/emcstfe/models/response/GetMovementResponseSpec.scala]: Updated test and pushed another commit

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢

**Pull request merged**